cpp/ycm/IdentifierUtils.cpp

```diff
@@ -37,9 +36,9 @@ const char *const TAG_REGEX =
   // The second field is the path to the file that has the identifier; either
   // absolute or relative to the tags file.
   "([^\\t\\n\\r]+)"
```

Comment:
You changed `ReadUtf8File` to return a vector of lines, thus you do not have to check for `\r` and `\n` anymore.